### ISSUE TITLE
NodeJS db-mysql Window Install

### ISSUE THREAD

**Issue by weilies** [mariano/node-db-mysql#89]
Good day support,

i am pretty happy with NodeJS, but something stop me when i tried to install db-mysql
# console

C:\Documents and Settings\Administrator\Desktop\temp\nodejs>npm install db-mysql

npm http GET https://registry.npmjs.org/db-mysql
npm http 304 https://registry.npmjs.org/db-mysql

> db-mysql@0.7.6 install C:\Documents and Settings\Administrator\Desktop\temp\no
> dejs\node_modules\db-mysql
> node-waf configure build

'node-waf' is not recognized as an internal or external command,
operable program or batch file.

> db-mysql@0.7.6 preuninstall C:\Documents and Settings\Administrator\Desktop\te
> mp\nodejs\node_modules\db-mysql
> rm -rf build/*

'rm' is not recognized as an internal or external command,
operable program or batch file.
npm WARN continuing anyway db-mysql@0.7.6 preuninstall: `rm -rf build/*`
npm WARN continuing anyway `cmd "/c" "rm -rf build/*"` failed with 1
npm ERR! db-mysql@0.7.6 install: `node-waf configure build`
npm ERR! `cmd "/c" "node-waf configure build"` failed with 1
npm ERR!
npm ERR! Failed at the db-mysql@0.7.6 install script.
npm ERR! This is most likely a problem with the db-mysql package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-waf configure build
npm ERR! You can get their info via:
npm ERR!     npm owner ls db-mysql
npm ERR! There is likely additional logging output above.

npm ERR! System Windows_NT 5.1.2600
npm ERR! command "C:\Program Files\nodejs\\node.exe" "C:\Program Files\nod
ejs\node_modules\npm\bin\npm-cli.js" "install" "db-mysql"
npm ERR! cwd C:\Documents and Settings\Administrator\Desktop\temp\nodejs
npm ERR! node -v v0.8.14
npm ERR! npm -v 1.1.65
npm ERR! code ELIFECYCLE
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     C:\Documents and Settings\Administrator\Desktop\temp\nodejs\npm-deb
ug.log
# npm ERR! not ok code 0

Is there an easy way to install mysql for window user?

Thanks

**Comment by NachoSoto** [comment]
Same here on Mac OS X

**Comment by pelister** [comment]
I think those commands were for linux based environments, I still dont know how to install it on Windows - mine is windows XP

**Comment by tdicusmurray** [comment]
has this problem been solved?

**Comment by weilies** [comment]
i manage to resolve it via a fresh install in Mac, Window.... erm.. i gave up

**Comment by Shajithk** [comment]
Yes, I got the same error while I tried to install in windows 7 OS.

Error Message below:
/*****************************************************************************************/
'node-waf' is not recognized as an internal or external command,
operable program or batch file.

> db-mysql@0.7.6 preuninstall E:\Program Files\nodejs\node_modules\db-mysql
> rm -rf build/*

'rm' is not recognized as an internal or external command,
operable program or batch file.
npm WARN continuing anyway db-mysql@0.7.6 preuninstall: `rm -rf build/*`
npm WARN continuing anyway Exit status 1
npm ERR! db-mysql@0.7.6 install: `node-waf configure build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the db-mysql@0.7.6 install script.
npm ERR! This is most likely a problem with the db-mysql package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-waf configure build
npm ERR! You can get their info via:
npm ERR!     npm owner ls db-mysql
npm ERR! There is likely additional logging output above.

npm ERR! System Windows_NT 6.1.7600
npm ERR! command "E:\Program Files\nodejs\\node.exe" "E:\Program Files\nod
ejs\node_modules\npm\bin\npm-cli.js" "install" "db-mysql"
npm ERR! cwd E:\Program Files\nodejs
npm ERR! node -v v0.10.29
npm ERR! npm -v 1.4.14
npm ERR! code ELIFECYCLE
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     E:\Program Files\nodejs\npm-debug.log
npm ERR! not ok code 0

**Comment by GTrebaol** [comment]
UP. Can't run a node.js application with my-sql driver. It would require all the people involved on the project to run on unix based OS...
